cpp/ql/lib/semmle/code/cpp/models/implementations/Iterator.qll

```diff
@@ -150,24 +157,28 @@ private class IteratorCrementOperator extends Operator, DataFlowFunction {
 /**
  * A non-member `operator+` function for an iterator type.
  */
-private class IteratorAddOperator extends Operator, TaintFunction {
+class IteratorAddOperator extends Operator {
   FunctionInput iteratorInput;
 
   IteratorAddOperator() {
     this.hasName("operator+") and
     iteratorInput = getIteratorArgumentInput(this, [0, 1])
   }
 
+  FunctionInput getIteratorInput() { result = iteratorInput }
```

Comment:
This and the others like it should get the same treatment as `IteratorCrementOperator.getIteratorInput()`, whatever exactly we decide to do with it.

Reply:
Fixed in 7583fe2ad850773fd3939b36ff9292dabbf6a7c6.